website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -78,22 +88,43 @@ Options:
       --network-file <path>     Networks config file path (default: "./networks.json")
 ```
 
-The `add` command will fetch the ABI from Etherscan (unless an ABI path is specified with the `--abi` option), and will create a new `dataSource` in the same way that `graph init` command creates a `dataSource` `--from-contract`, updating the schema and mappings accordingly.
+#### Specifics
 
-The `--merge-entities` option identifies how the developer would like to handle `entity` and `event` name conflicts:
+The `add` command will fetch the ABI from Etherscan unless an ABI path is specified with the `--abi` option. Subsequently, it will create a new `dataSource` like `graph init` command creates a dataSource `--from-contract` and will update the schema and mappings accordingly. This allows you to index implementation contracts from their proxy contracts.
+
+- The `--merge-entities` option identifies how the developer would like to handle `entity` and `event` name conflicts:
 
 - If `true`: the new `dataSource` should use existing `eventHandlers` & `entities`.
-- If `false`: a new entity & event handler should be created with `${dataSourceName}{EventName}`.
 
-The contract `address` will be written to the `networks.json` for the relevant network.
+- If `false`: a new `entity` & `event` handler should be created with `${dataSourceName}{EventName}`.
+
+- The contract `address `will be written to the `networks.json` for the relevant network.
+
+> Note: When using the interactive cli, after successfully running `graph init`, you'll be prompted to add a new `dataSource`.
+
+## Components of a Subgraph
+
+### The Subgraph Manifest
+
+The subgraph manifest, `subgraph.yaml`, defines the smart contracts your subgraph will index, the events from these contracts to pay attention to, and how to map event data to entities that Graph Node stores and allows to query.
+
+The **subgraph definition** consists of the following files:
+
+- `subgraph.yaml`: Contains the subgraph manifest
+
+- `schema.graphql`: a GraphQL schema defining the data stored for your subgraph and how to query it via GraphQL.
+
+- `mapping.ts`: [AssemblyScript Mappings](https://github.com/AssemblyScript/assemblyscript) code that translates event data into entities defined in your schema (e.g. `mapping.ts` in this guide)
```

Comment:
There are inconsistencies between these 3 bullet points:
- Two of them have a capital letter after the `:` (though for `AssemblyScript` it could just be because it's a proper noun), while the other one has a lowercase letter.
- The same two bullet points don't have a period at the end of their sentence, while the other one does.